Upstream:no

Our musl version doesn't expose the SEEK_DATA and SEEK_HOLE in `unistd.h`...
Only version >= 1.2.3 do that.
---
 src/shared/copy.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/shared/copy.h b/src/shared/copy.h
--- a/src/shared/copy.h
+++ b/src/shared/copy.h
@@ -92,3 +92,11 @@ int copy_xattr(int df, const char *from, int dt, const char *to, CopyFlags copy_
 
 int reflink(int infd, int outfd);
 int reflink_range(int infd, uint64_t in_offset, int outfd, uint64_t out_offset, uint64_t sz);
+
+#ifndef SEEK_DATA
+#define SEEK_DATA 3
+#endif
+
+#ifndef SEEK_HOLE
+#define SEEK_HOLE 4
+#endif
